taxonomies/categories.txt

```diff
@@ -81899,6 +81907,7 @@ wikidata:en:Q600396
 # Fish patties are generally called "fish burgers", but it is not put as a synonym since it could be confused with "fish hamburgers"
 <en:Fishes
 en:Fish patties
+fr:Galettes de poisson
```

Comment:
```suggestion
fr:Galettes au poisson
```

Reply:
@stephanegigandet @moon-rabbitOFF you'll have a duplicate with :
><en:Meals
en:Crepes filled with fish, Buckwheat crepes filled with fish
fr:Crêpes au poisson, Galettes au poisson, Crêpes fourrées au poisson, Galettes fourrées au poisson
agribalyse_food_code:en:25625
ciqual_food_code:en:25625
ciqual_food_name:en:Crepe or buckwheat crepe, filled with fish and/or seafood
ciqual_food_name:fr:Crêpe ou Galette fourrée au poisson et / ou fruits de mer